Source/Platform/Assert.cpp

```diff
@@ -13,6 +13,7 @@
 using namespace Helium;
 
 volatile int32_t Assert::sm_active = 0;
```

Comment:
I don't think either of these should be volatile.

Reply:
Yes, you are right.